presto-cli/src/main/java/com/facebook/presto/cli/TsvPrinter.java

```diff
@@ -55,6 +55,13 @@ public void printRows(List<List<?>> rows, boolean complete)
     public void finish()
             throws IOException
     {
+        flush();
+    }
+
+    @Override
+    public void flush()
+            throws IOException
+    {
         printRows(ImmutableList.<List<?>>of(), true);
```

Comment:
As I understand flush, flushes all the written data to a kind of a stream and it should not write anything new to buffers and then eventually to a stream. Though, it is totally fine that `printRows()` flushes internally.

I do not want to block or even disturb your work. I am just asking out of a curiosity and due learning the presto codebase.